sci-chemistry/openbabel/openbabel-9999.ebuild

```diff
@@ -107,51 +93,18 @@ prepare_python_bindings() {
 src_prepare() {
 	cmake_src_prepare
 
-	if use perl; then
-		perl_set_version
+	# Prevent bundled inchi as fallback
+	rm -r include/inchi || die
 
-		sed -e "/\${LIB_INSTALL_DIR}\/auto/s|\${LIB_INSTALL_DIR}|${VENDOR_ARCH}|" \
-			-e "/\${LIB_INSTALL_DIR}\/Chemistry/s|\${LIB_INSTALL_DIR}|${VENDOR_ARCH}|" \
-			-i scripts/CMakeLists.txt || die
-	fi
-
-	if use python; then
-		# Skip the python bindings sections as we'll append our own
-		sed -e '/^if (PYTHON_BINDINGS)$/s|PYTHON_BINDINGS|false|' \
-			-i {scripts,test}/CMakeLists.txt || die
-		if use test; then
-			# Problems with testbindings built with -O2
-			local test_skip="@unittest.skip('Similar to Issue #2138')"
-			sed -e "/def testTemplates/s|^|    ${test_skip}\\n|" \
-				-i test/testbindings.py || die
-			test_skip="@unittest.skip('Similar to Issue #2246')"
-			sed -e "/^def test_write_string/s|^|${test_skip}\\n|" \
-				-i test/testobconv_writers.py || die
-			fi
-		python_foreach_impl prepare_python_bindings
-	fi
-
-	# Remove dependency automagic
-	if ! use png; then
-		sed -e '/^find_package(Cairo/d' -i CMakeLists.txt || die
-	fi
-	if ! use wxwidgets; then
-		sed -e '/^find_package(wxWidgets/d' -i CMakeLists.txt || die
-	fi
-	if ! use inchi; then
-		sed -e '/^else()$/s|else\(\)|elseif\(false\)|' \
-			-i cmake/modules/FindInchi.cmake || die
-	fi
-
-	# Don't install example bins to /usr/bin
-	if use examples; then
-		sed -e "/RUNTIME DESTINATION/s|bin|share/doc/${PF}/examples|" \
-			-i doc/examples/CMakeLists.txt || die
-	fi
+	# Problems with testbindings built with -O2
```

Comment:
That sounds quite worrying and should be investigated at some point.

Reply:
OK. I'll check this. A test passed with -O3 should be sufficient ?

Reply:
Yes, that should be OK. Maybe check with LTO as well and Clang -O2.

Reply:
Let's drop the skip and then if issues arise in a bug, we can investigate properly. Can you check `git log` to see when this got added?

Reply:
Added with last snapshot: openbabel-3.1.1_p20210225

Reply:
0001-diff-implement-config.diff.renames-copies-harder.patch: https://paste.gentoo.zip/LpVnm7vK. I may add it into `dev-vcs/git` soon.

As for the change itself, can you mention in the commit message that it was added in that commit but at least seems fixed now (checked with ...) so dropping it, and we can investigate if some report shows up?

Reply:
Done